src/components/MerchantsPage/DonationPool.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { SET_MODAL_VIEW } from '../../utilities/hooks/ModalPaymentContext/constants';
+import { useModalPaymentDispatch } from '../../utilities/hooks/ModalPaymentContext/context';
+import styled from 'styled-components';
+import Modal from '../Modal';
+import Hero from './images/hero.png';
+import {
+  phoneScreens,
+  tabletScreens,
+} from '../../utilities/general/responsive';
+
+const ModalBox: any = Modal;
+
+const DonationPoolBox = () => {
+  const dispatch = useModalPaymentDispatch();
+
+  const openModal = (e: any) => {
+    e.preventDefault();
+    console.log('made it here');
+    dispatch({ type: SET_MODAL_VIEW, payload: 0 });
+  };
+
+  return (
+    <React.Fragment>
+      <Container>
+        <ColumnContainer>
+          <h4>
+            Checked out our merchants but not sure who to donate to first?
+          </h4>
+          <span>You can support by donating to our donation pool!</span>
+          <span>
+            All donations will be distributed evenly to our merchants.
+          </span>{' '}
+          <br />
+          <button className="button--red-filled" onClick={openModal}>
+            SUPPORT CHINATOWN
```

Comment:
Thoughts on making the button text something like "MAKE A POOL DONATION" or something that is more direct about what action you're taking when you click the button?

Reply:
cc @nanxiy but I do like the smaller text

Reply:
“Donate to Chinatown”? Idk something shorter is better esp for mobile